src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,85 @@
-const Countdown = () => null;
+import { useEffect, useState } from 'react';
+import ActionButton from '../generic/ActionButton';
+import DisplayWindow from '../generic/DisplayWindow';
+import InputField from '../generic/Input';
+import Loading from '../generic/Loading';
+
+const Countdown = () => {
+    //States being maintained
+    const [time, setTime] = useState(0);
+    const [minutes, setMinutes] = useState(0);
+    const [seconds, setSeconds] = useState(0);
+    const [isRunning, setIsRunning] = useState(false);
+
+    useEffect(() => {
+        let timer: number | undefined;
+        //if statement to run countdown and check if time has reached 0
+        if (isRunning && time > 0) {
+            timer = setInterval(() => {
```

Comment:
setInterval returns an interval that repeats every X ms. If you are going to clear the interval and recreate it each time, there's no point in using an interval but rather setTimeout, which runs just once.